### PR TITLE
add PR chain example: for main branch

### PR DESCRIPTION
**Pull request by sakka2**
This is an example of chain PR feature.

### CODE REVIEW

**Review by s-yoshika — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+        requires: [ ~pr, ~commit ]
+        steps:
+            - echo: echo "this is first job."
+    second-job:
+        requires: [ first-job ]
+        steps:
+            - echo: echo "this is second job."
+```
+
+Then, if you make a PR, subsequent jobs in PR branch will be chained according to workflow in PR evnet.
```
Comment (tk3fftk) [inline README.md]: evnet -> event

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
-# chain-pr-example
\ No newline at end of file
+# Chain PR Example
+
+To use chain PR feature in Screwdriver v4, set `true` in pipeline-level annotation `screwdriver.cd/chainPR`.
```
Comment (tk3fftk) [inline README.md]: Screwdriver V4 => Screwdriver.cd

File: README.md [hunk 3]
```diff
@@ -1 +1,52 @@
...
+```
+
+Then, if you make a PR, subsequent jobs in PR branch will be chained according to workflow in PR evnet.  
+In this example, `second-job` will executed after the `first-job`, and `third-job` will be executed after the `second-job`.
+```
+# screwdriver.yaml in PR branch
+
+shared:
+    image: node:8
+
```
Comment (tk3fftk) [inline README.md:38]: I think missing annotations is looks unnatural in PR.

File: README.md [hunk 4]
```diff
@@ -1 +1,52 @@
...
+This annotation must be configured in ***main branch***.
+
+```
+# screwdriver.yaml in main branch
+
+shared:
+    image: node:8
+
+annotations:
+  screwdriver.cd/chainPR: true
```
Comment (wdstar) [inline README.md]: indent not enough.

File: README.md [hunk 5]
```diff
@@ -1 +1,52 @@
...
+        steps:
+            - echo: echo "this is first job."
+    second-job:
+        requires: [ first-job ]
+        steps:
+            - echo: echo "this is second job."
+```
+
+Then, if you make a PR, subsequent jobs in PR branch will be chained according to workflow in PR evnet.  
+In this example, `second-job` will executed after the `first-job`, and `third-job` will be executed after the `second-job`.
```
Comment (wdstar) [inline README.md]: `second-job`  will **be** executed

File: README.md (hunk 1 above)
Comment (sakka2) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (sakka2) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (sakka2) [reply README.md:38]: fixed

File: README.md (hunk 4 above)
Comment (sakka2) [reply README.md]: fixed

File: README.md (hunk 5 above)
Comment (sakka2) [reply README.md]: fixed

**Review by wdstar — APPROVED** [review]
LGTM

File: README.md (hunk 3 above)
Comment (tk3fftk) [reply README.md:38]: We should also fix https://github.com/screwdriver-cd-test/chain-pr-example/pull/2

Comment (sakka2) [reply README.md:38]: fixed #2

**Review by tk3fftk — APPROVED** [review]
👍

**Pull request merged**